event_session/models/event_session.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 David Vidal<david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class EventSession(models.Model):
+    _name = 'event.session'
```

Comment:
What's the point on creating this model instead of using upstream `event.track`?